include/eosio/vm/signals.hpp

```diff
@@ -18,7 +18,13 @@ namespace eosio { namespace vm {
    inline thread_local std::atomic<sigjmp_buf*> signal_dest{nullptr};
 
    __attribute__((visibility("default")))
-   inline thread_local std::vector<std::span<std::byte>> protected_memory_ranges;
+   inline thread_local std::span<std::byte> code_memory_range;
```

Comment:
Why not just name `code_memory_range` as `code_range`?

Reply:
In this case I think it mirrors the naming of existing `get_*_span()` calls. `s/_span$/_memory_range/` (i.e. keeping the prefix in tact)